back/lib/participation_method/native_survey.rb

```diff
@@ -59,6 +59,10 @@ def default_fields(custom_form)
       ]
     end
 
+    def allowed_extra_field_input_types
```

Comment:
I don't fully understand how we could apply these to the printable and importable fields? Because the lists of input_types are different and the `IdeaCustomFieldsService` seems to use a mix of extra fields and built-in fields. But if you think there's a way to apply this there, then I would like to do it. Or if you think it's more efficient, we could continue our discussion in a short call?